src/test/java/edu/wpi/grip/core/serialization/SerializationTest.java

```diff
@@ -17,8 +17,8 @@
 import static org.junit.Assert.assertEquals;
 
 public class SerializationTest {
-    private final EventBus eventBus = new EventBus();
-    private final Serialization serialization = new Serialization(eventBus);
+/*    private final EventBus eventBus = new EventBus();
```

Comment:
Um, no?  I'm almost completely re-writing the serialization stuff, so these tests probably won't even compile.  When this is ready for review, all amend it with real tests.